spec/rubocop/cop/variable_force_spec.rb

```diff
@@ -21,7 +21,17 @@
     end
 
     context 'when processing an empty regex' do
-      let(:node) { s(:match_with_lvasgn, s(:regexp, s(:regopt)), s(:str)) }
+      let(:node) { parse_source('// =~ ""').ast }
+
+      it 'does not raise an error' do
+        expect { force.process_node(node) }.not_to raise_error
+      end
+    end
+
+    # FIXME: Remove `broken_on: jruby` when JRuby incompatible is resolved:
+    # https://github.com/jruby/jruby/issues/7113
+    context 'when processing a regex with regopt', broken_on: :jruby do
```

Comment:
This new test will be skipped due to JRuby incompatibility.
https://github.com/jruby/jruby/issues/7113